Makefile

```diff
@@ -100,33 +107,39 @@ gosec: scripts/go/bin/gosec
 		-conf=./scripts/go/configs/gosec.json \
 		$(GO_FILES)
 
-golangci-lint: scripts/go/bin/golangci-lint
-	@echo "lint via golangci-lint"
-	@scripts/go/bin/golangci-lint run \
-		--config ./scripts/go/configs/.golangci.yml \
-		$(GO_FILES)
-
 go-vet:
 	@echo "lint via go vet"
 	@$(GO) vet $(GO_FILES)
 
-lint-go: go-vet golangci-lint revive revive-alerting gosec
+lint-go: go-vet golangci-lint revive revive-alerting gosec ## Run all code checks for backend.
 
 # with disabled SC1071 we are ignored some TCL,Expect `/usr/bin/env expect` scripts
-shellcheck: $(SH_FILES)
+shellcheck: $(SH_FILES) ## Run checks for shell scripts.
 	@docker run --rm -v "$$PWD:/mnt" koalaman/shellcheck:stable \
 	$(SH_FILES) -e SC1071 -e SC2162
 
-run: scripts/go/bin/bra
-	@scripts/go/bin/bra run
+##@ Docker
+
+build-docker-dev: ## Build Docker image for development (fast).
+	@echo "build development container"
+	@echo "\033[92mInfo:\033[0m the frontend code is expected to be built already."
+	$(GO) run build.go -goos linux -pkg-arch amd64 ${OPT} build pkg-archive latest
+	cp dist/grafana-latest.linux-x64.tar.gz packaging/docker
+	cd packaging/docker && docker build --tag grafana/grafana:dev .
+
+build-docker-full: ## Build Docker image for development.
+	@echo "build docker container"
+	docker build --tag grafana/grafana:dev .
+
+##@ Development environment
 
 # create docker-compose file with provided sources and start them
 # example: make devenv sources=postgres,openldap
 ifeq ($(sources),)
 devenv:
 	@printf 'You have to define sources for this command \nexample: make devenv sources=postgres,openldap\n'
 else
-devenv: devenv-down
+devenv: devenv-down ## Set up development environment.
```

Comment:
> Set up development environment

This seems like something you need to do in order to develop. 
Maybe something like this:

Start optional external dependencies via docker-compose like postgres, mysql, prometheus, openldap, elasticsearch etc.

Or

Start optional services of postgres, mysql, prometheus, openldap, elasticsearch, etc (you choose)